src/main/java/com/google/googleidentity/user/UserDetailsService.java

```diff
@@ -21,23 +21,37 @@
 /** UserDetailsService Interface. Support basic user information access and modification */
 public interface UserDetailsService {
 
-  /** @return the user with the username */
+  /**
+   * @param username
```

Comment:
If there is no extra information to provide, then don't need this @param. Same for others.